source/libs/features.tsx

```diff
@@ -98,24 +91,19 @@ const globalReady: Promise<GlobalOptions> = new Promise(async resolve => {
 	document.documentElement.classList.add('refined-github');
 
 	// Options defaults
-	const options: GlobalOptions = {
-		disabledFeatures: '',
-		customCSS: '',
-		logging: false,
-		...await new OptionsSync().getAll()
-	};
-
-	if (options.customCSS.trim().length > 0) {
+	const options = await new OptionsSync().getAll();
```

Comment:
```suggestion
	const options = (await new OptionsSync().getAll()) as Options;
```

This `as` can probably be avoided if we improve the types for `OptionsSync`.

Reply:
I wasn't sure how to suggest changes for multiple lines, hence the 3 one liner suggested changes. 🙃

Reply:
This just moves the `as Options` but I'd rather avoid it

Reply:
Moving it prevents a cast to `string` when trimming. What would be interesting is to improve the types of `OptionsSync`. Making `getAll` generic, would avoid any casting.

Reply:
I see what you mean now.

> Making getAll generic, would avoid any casting

👍